src/buffer/bufferv1.ts

```diff
@@ -0,0 +1,37 @@
+// @ts-check
+import { SenderOptions } from "../options";
+import { SenderBuffer } from "./index";
+import { SenderBufferBase } from "./base";
+
+/**
+ * Buffer implementation for protocol version 1.
+ * Sends floating point numbers in their text form.
+ */
+class SenderBufferV1 extends SenderBufferBase {
+  constructor(options: SenderOptions) {
+    super(options);
+  }
+
+  /**
+   * Writes a float column with its value into the buffer using v1 serialization (text format).
```

Comment:
nit: we should probably mention that this method writes 64-bit FP number and that it works for `double` and `float` database types.